book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
> - [OPTIONAL] I suggest you use the shortcut for Getters and Setters `attr_accessor` instead of using `attr_reader` for each. You can check this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor) to the lesson you took.👍